src/palo-alto-networks/azext_palo_alto_networks/aaz/latest/palo_alto/cloudngfw/firewall/_get_log_profile.py

```diff
@@ -0,0 +1,252 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "palo-alto cloudngfw firewall get-log-profile",
```

Comment:
Usually we will use `show` instead of `get`
```suggestion
    "palo-alto cloudngfw firewall show-log-profile",
```